src/main/java/microsoft/exchange/webservices/data/attribute/Attachable.java

```diff
@@ -31,7 +31,6 @@
 /**
  * The Interface Attachable.
  */
-@Target(ElementType.TYPE)
-@Retention(RetentionPolicy.RUNTIME) public @interface Attachable {
+@Target(ElementType.TYPE) @Retention(RetentionPolicy.RUNTIME) public @interface Attachable {
```

Comment:
Annotation style is incorrect (very unusual) everywhere. Should be one per line, above the block.
https://google-styleguide.googlecode.com/svn/trunk/javaguide.html#s4.8.5-annotations